### PR TITLE
feat(medusa): Cart custom query strategy

### PR DESCRIPTION
**Pull request by olivermrbl**

### CODE REVIEW

File: packages/medusa/src/models/cart.ts [hunk 1]
```diff
@@ -390,6 +391,13 @@ export class Cart extends SoftDeletableEntity {
   gift_card_total?: number
   gift_card_tax_total?: number
 
+  @AfterLoad()
+  private afterLoad(): void {
```
Comment (olivermrbl) [inline packages/medusa/src/models/cart.ts:395]: comment: We can now bring this back, as we are no longer using the query strategy.

**Review by riqwan — APPROVED** [review]
Looks gucci to me. 🎉

**Review by pKorsholm — APPROVED** [review]
LGTM

**Review by adrien2p — APPROVED** [review]
LGTM

**Pull request merged**